runtime-modules/forum/src/mock.rs

```diff
@@ -603,19 +603,17 @@ pub fn good_poll_alternative_text() -> Vec<u8> {
     b"poll alternative".to_vec()
 }
 
-pub fn generate_poll(
+pub fn generate_poll_input(
```

Comment:
Add comments for public methods.

Reply:
Good point, done https://github.com/Joystream/joystream/pull/2531/commits/4bb08fdb7c9a5c49976eb313b279b62aa8973780